src/components/FindMovie/FindMovie.tsx

```diff
@@ -16,42 +80,58 @@ export const FindMovie: React.FC = () => {
               type="text"
               id="movie-title"
               placeholder="Enter a title to search"
-              className="input is-danger"
+              className={classNames('input', { 'is-danger': isError })}
+              value={title}
+              onChange={handleOnChangeTitle}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {isError && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={classNames(
+                'button is-light',
+                {
+                  'is-loading': isLoading,
+                },
+              )}
+              onClick={() => handleFindMovie()}
+              disabled={title.trim().length === 0}
             >
-              Find a movie
+              {moviePreview !== null ? 'Search again' : 'Find a movie'}
             </button>
           </div>
 
-          <div className="control">
-            <button
-              data-cy="addButton"
-              type="button"
-              className="button is-primary"
-            >
-              Add to the list
-            </button>
-          </div>
+          {isMoviePreview && (
```

Comment:
You don't really need a separate variable for it 
```suggestion
          {moviePreview && (
```